charts/pcq-frontend/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: A Helm chart for the HMCTS PCQ product
 name: pcq-frontend
 home: https://github.com/hmcts/pcq-frontend
-version: 1.0.31
+version: 1.1.0
```

Comment:
Make any changes to the `/charts` folder in a separate PR, get that approved and merged first.
This is to avoid race conditions in Flux.